pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -666,12 +676,16 @@ func (q *PacketInQueue) AddOrDrop(packet *ofctrl.PacketIn) bool {
 		return true
 	default:
 		// Channel is full.
+		q.pktDrops++
```

Comment:
This is probably a good candidate for a metrics time series, but I also understand that a log message is easier to troubleshoot :)

Reply:
Thanks for the advice, will consider a metric too.

Reply:
@antoninbas while testing https://github.com/antrea-io/antrea/pull/7242 and https://github.com/antrea-io/antrea/pull/7243, I found this process never dropped any packets, so I'll skip adding a metric for it until we start observing drops from the logs.